transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/CustomAttributesStoreImpl.kt

```diff
@@ -30,10 +40,14 @@ internal class CustomAttributesStoreImpl(
     }
 
     private fun isCustomAttributesValid(customAttributes: Map<String, String>): Boolean {
-        if (customAttributes.isEmpty() || this.customAttributes == customAttributes) {
-            log.w { LogMessages.CUSTOM_ATTRIBUTES_EMPTY_OR_SAME }
-            return false
-        } else if (isSizeExceeded(customAttributes)) {
+        return maxCustomDataBytes.isUnset() || (
```

Comment:
The return condition of this function looks quite difficult for reading. 
I think we can sacrifice a couple of lines of code to make it more readable for human eye